File: demo/java-demo/src/test/java/com/alibaba/demo/one2multi/CSvcMock.java
package com.alibaba.demo.one2multi;

import com.alibaba.testable.core.annotation.MockInvoke;

public class CSvcMock {

    @MockInvoke(targetClass = String.class, targetMethod = "format")
    public String c_format(String format, Object... args) {
        return "c_mock";
    }

}

File: demo/kotlin-demo/src/test/kotlin/com/alibaba/demo/one2multi/CSvcMock.kt
package com.alibaba.demo.one2multi

import com.alibaba.testable.core.annotation.MockInvoke

class CSvcMock {

    @MockInvoke(targetClass = String::class, targetMethod = "format")
    fun c_format(format: String, vararg args: Any?): String {
        return "c_mock"
    }

}
